test/src/utils/mine.rs

```diff
@@ -0,0 +1,70 @@
+use crate::Node;
+use ckb_types::core::{BlockBuilder, BlockView};
+use ckb_types::packed;
+
+pub fn mine_until_out_ibd_mode(node: &Node) {
+    mine_until_bool(node, || node.get_tip_block_number() > 0)
+}
+
+/// The `[1, PROPOSAL_WINDOW.farthest()]` of chain is called as bootstrap period. Cellbases w
+/// this period are zero capacity.
+///
+/// This function will generate blank blocks until node.tip_block_number > PROPOSAL_WINDOW.fa
+///
+/// Typically involve this function at the beginning of test.
+pub fn mine_until_out_bootstrap_period(node: &Node) {
+    let farthest = node.consensus().tx_proposal_window().farthest();
```

Comment:
there have a `DEFAULT_TX_PROPOSAL_WINDOW` in `test/src/lib.rs`, maybe we can init this in the beginning of case execution, or just directly use the default value of this variable.

Reply:
Oh, I think the`DEFAULT_TX_PROPOSAL_WINDOW` in integration-test is a legacy and redundant constant, I just intend to remove it. `node.consensus().tx_proposal_window()` returns the most realistic value.